maligned.go

```diff
@@ -22,10 +23,15 @@ var fset = token.NewFileSet()
 func main() {
 	flag.Parse()
 
+	importPaths := gotool.ImportPaths(flag.Args())
+	if len(importPaths) == 0 {
+		return
+	}
```

Comment:
I'm not sure why I added the if statement here. According to [its documentation](https://godoc.org/github.com/kisielk/gotool#ImportPaths), it can never happen:

> If args is empty, a single element "." is returned.